app/views/shared/_parole_info.html.erb

```diff
@@ -22,13 +22,13 @@
         <td id='parole-eligibility-date' class="govuk-table__cell table_cell__left_align  govuk-!-width-one-half"><%= format_date(@prisoner.parole_eligibility_date) %></td>
       </tr>
     <% end %>
-    <% if @prisoner.target_hearing_date.present? %>
+    <% if @prisoner.current_parole_record&.target_hearing_date.present? %>
       <tr class="govuk-table__row">
         <td class="govuk-table__cell govuk-!-width-one-half">Target hearing date</td>
         <td id='target-hearing-date' class="govuk-table__cell table_cell__left_align  govuk-!-width-one-half"><%= format_date(@prisoner.target_hearing_date) %></td>
       </tr>
     <% end %>
-    <% if @prisoner.current_parole_record.hearing_outcome.present? %>
+    <% if @prisoner.target_hearing_date.present? && @prisoner.target_hearing_date <= Time.zone.today && @prisoner.current_parole_record&.hearing_outcome.present?%>
```

Comment:
Maybe another method on MpcOffender, or at least a view helper? The name of that method would act as documentation to anyone reading this in the future.